lldb/test/Shell/Swift/Inputs/No.swiftmodule-ObjC.swift

```diff
@@ -13,10 +13,9 @@ func f() {
   // The Objective-C runtime recognizes this as a tagged pointer.
   // CHECK-DAG: (NSNumber) inlined = {{.*}}Int64(42)
   let inlined = NSNumber(value: 42)
-  // FIXME: The dataformatters wrongly think this is an OptionSet.
```

Comment:
I'm not sure what to do about this comment. It's still being formatted by `SwiftOptionSetSummaryProvider`, but the test output is what would be expected. Maybe there's a separate bug/report to reference instead.

Reply:
Let's just delete it. Since the output is going to be indistinguishable, having the comment in the test makes little sense, and we'll just call it out in the code instead.